src/routes/user.js

```diff
@@ -1,9 +1,10 @@
 const express = require('express');
 const userController = require('../controllers/user');
+const authorize = require('../middleware/guard');
 
 const router = express.Router();
 
 router.get('/profile', userController.getProfile);
 router.get('/:id', userController.getSingleUser);
-
+router.delete('/delete', authorize, userController.deleteProfile);
```

Comment:
Doesn't the endpoint automatically check if a user is authorized or not? I'm only asking because I'm not quite sure.

Reply:
- PR looks good. Only thing that can be fixed is the documentation: it isn't functional.
- Can you please change the endpoint name to `/profile/terminate` or `/profile/delete`, in both routes and docs?

Reply:
I also think we dont need the middleware here since we fixed the other middleware with the methods and paths